src/ngraph/pass/opset1_transform.cpp

```diff
@@ -68,6 +68,11 @@ bool pass::Opset1Transformation::run_on_node(shared_ptr<Node> node)
                  op_version,
                  " found.");
 
+// Not all enumeration values not explicitly handled in switch
```

Comment:
Double negation in comment. Change to:
`// Not all enumeration values explicitly handled in switch`

Reply:
Done